File: 1064.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var s = 0
var m = 0
for(var c = 1; c < 7; c++){
    ca = Number(lines.shift());
    if (ca > 0){
        var s = s + 1
        var m = m + ca}}
console.log(`${s} valores positivos
${(m/s).toFixed(1)}`)

File: 1064.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        double m = 0;
        int s = 0;
        for(int c = 1; c < 7; c++) {
                double p = input.nextDouble();
                if(p > 0) {
                        s += 1;
                        m += p;
                }
        }
        System.out.println(s+" valores positivos");
        System.out.printf("%.1f%n", m/s);
}
}
